src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/FindBreakpointEvidenceSpark.java

```diff
@@ -451,8 +453,8 @@ public EvidenceScanResults(final ReadMetadata readMetadata,
                         new ReadsForQNamesFinder(broadcastQNamesMultiMap.value(), nIntervals,
                                 includeMappingLocation, readItr, filter).iterator(), false)
                 .combineByKey(x -> x,
```

Comment:
I refactored/moved the `combineLists` from FindBreakpointEvidence to SVFastqUtils, so would rather let Ted handle this as he has many acrobatic tricks on how to avoid shuffles, and I refrain from messing with that.